client/devdocs/design/index.jsx

```diff
@@ -101,6 +102,7 @@ export default React.createClass( {
 					<InfoPopover />
 					<InputChrono />
 					<Notices />
+					<NoticesGlobal />
```

Comment:
Not totally introduced in this PR, but the naming here is a little confusing: `Notices` is the example component for `Notice`, `NoticesGlobal` is the example component for `Notices`, even though we have a separate component named `GlobalNotices`.